store/commitment/iavl/tree.go

```diff
@@ -83,6 +93,140 @@ func (t *IavlTree) Prune(version uint64) error {
 	return t.tree.DeleteVersionsTo(int64(version))
 }
 
+// Snapshot implements snapshottypes.CommitSnapshotter. The snapshot output for a given format must be
+// identical across nodes such that chunks from different sources fit together.
```

Comment:
We can avoid the godocs as long as the `CommitSnapshotter` interface defines the godoc. This saves space.